content-review/impl/vericite/src/main/java/org/sakaiproject/contentreview/vericite/client/model/ExternalContentUploadInfo.java

```diff
@@ -105,6 +108,24 @@ public void setFilePath(String filePath) {
     this.filePath = filePath;
   }
 
+  public ExternalContentUploadInfo headers(Object headers) {
+    this.headers = headers;
+    return this;
+  }
+
+   /**
+   * A map of required headers
+   * @return headers
+  **/
+  @ApiModelProperty(example = "null", value = "A map of required headers")
+  public Object getHeaders() {
+    return headers;
+  }
+
+  public void setHeaders(Object headers) {
```

Comment:
Object seems like a bad type for structured data. We are able to work with it just fine in string form, but you'll notice that we're doing some text manipulating. Removing curly brackets, splitting on a comma, splitting on equals sign, and setting a key and value. 

In the [api definition](https://github.com/vericite/LMS-API/blob/master/VeriCiteLmsApiV1.json#L104-L107), or a rest analog to a wsdl, the definition of the headers is that it is an object.

```
                "headers": {
                    "description": "A map of required headers",
                    "type": "object"
                },
```